### PR TITLE
[Tweaks] Slim ask answer notes properly

### PR DESCRIPTION
**Pull request by AprilSylph**
tumblr has an element within ask answer notes which from the class name looks like it's supposed to contain the answerer's response, but always contains nothing and messes up the activity dropdown when slimmed, so the tweak now also gets rid of it

| Before | After |
| :-----: | :----: |
| ugly | nice |

### CODE REVIEW

**Comment by nightpool** [comment]
maybe we can use :empty to make sure this doesn't have any effect if they
start filling in ask responses?

**Comment by AprilSylph** [comment]
apparently not since the blockquote contains two whole spaces

File: Extensions/tweaks.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //* TITLE Tweaks **//
-//* VERSION 5.7.1 **/
+//* VERSION 5.7.2 **/
```
Comment (nightpool) [inline Extensions/tweaks.js]: needs version update

Comment (AprilSylph) [reply Extensions/tweaks.js]: to reduce the number of force-pushes i need to do, i'd prefer you look at #1702 before i do this

**Pull request merged**